### DISCUSSION TITLE
Safari playback speed > 2x is very stutter / skips frames

### DISCUSSION THREAD

**Discussion by ireuben** [blakeblackshear/frigate#7726] [General]
On Safari (16.6 on MacOS Ventura) any playback speed over 2x starts dropping tons of frames, often skipping over seconds of realtime video. On Chrome on the same computer, looking at same video, even the 16x playback seems to be showing all original frames (looks super smooth -- just crazy fast!).

Having developed web-based video players in the past I understand this may be just be a browser quirk that is hard/impossible to work around, but thought I'd check if others are seeing the same issue, and if there are any codec/etc settings that might make my source video more compatible with Safari.

**Comment by NickM-27** [comment]
I see the same thing even across h265 & h264. Definitely just seems like a safari issue to me, not sure if there is something specific that would alleviate it

**Comment by boulete31** [comment]
Hi,
Same here.
If the web browser is webkit based (safari or orion) the playback is awful (lagging, frames dropping..). No problem with chrome.
